server/conf/application.conf

```diff
@@ -526,3 +527,7 @@ api_keys_ban_global_subnet = ${?CIVIFORM_API_KEYS_BAN_GLOBAL_SUBNET}
 # load balancer timeout.
 api_applications_list_max_page_size = 1000
 api_applications_list_max_page_size = ${?CIVIFORM_API_APPLICATIONS_LIST_MAX_PAGE_SIZE}
+
+# Feature flag toggles.
+status_tracking_enabled = true
```

Comment:
maybe would default to false here

Reply:
totally.  Just playing around atm.  Will fix up and send out formally.